package_scanner.nim

```diff
@@ -1,194 +1,218 @@
-# A very simple Nim package scanner.
-#
-# Scans the package list from this repository.
-#
-# Check the packages for:
-#  * Missing name
-#  * Missing/unknown method
-#  * Missing/unreachable repository
-#  * Missing tags
-#  * Empty tags
-#  * Missing description
-#  * Missing/unknown license
-#  * Insecure git:// url on GitHub
-#
-# Usage: nim r [-d:dontFetchRepos] package_scanner.nim 
-#
-# Copyright 2015 Federico Ceratto <federico.ceratto@gmail.com>
-# Released under GPLv3 License, see /usr/share/common-licenses/GPL-3
-
-import std/[httpclient, net, json, os, sets, strutils]
-
-const licenses = [
```

Comment:
What benefit is the warning supposed to provide? As long as there are no wrong values - we have not adopted any standard license identifiers - it only indicates we have not added the specific string to our list of known licenses. This list of know license identifiers is a lot of work to maintain and only serves as a way of shutting up the package scanner again.

Reply:
The license check only makes sense if someone maintains the list to check against. History has shown that nobody wants to do this and I am very reluctant to add back code that is destined to become unmaintained the moment it is added. If there is eventually a new concept that works I doubt it will benefit from a few lines of code that have been left in.

Reply:
Probably in the future, we might give some candidates based on edit distances if the license doesn't exist in the list.

Reply:
Ok